### PR TITLE
libecpint 1.0.7 (new formula)

### PR DESCRIPTION
**Pull request by junghans**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
For #132494

### CODE REVIEW

File: Formula/libecpint.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "-DLIBECPINT_BUILD_TESTS=OFF",
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    pkgshare.install "example/example.cpp"
+    pkgshare.install "example/hydrogen.bas"
+    pkgshare.install "example/iodine.bas"
```
Comment (carlocab) [inline Formula/libecpint.rb:26]:
```suggestion
    pkgshare.install "example/example.cpp",
                     "example/hydrogen.bas",
                     "example/iodine.bas"
```

File: Formula/libecpint.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  end
+
+  test do
+    system ENV.cxx, "-std=c++11", pkgshare/"example.cpp",
+      "-DHAS_PUGIXML", "-I#{include}", "-L#{lib}", "-lecpint",
+      "-o", "example"
```
Comment (carlocab) [inline Formula/libecpint.rb:32]:
```suggestion
    system ENV.cxx, "-std=c++11", pkgshare/"example.cpp",
                    "-DHAS_PUGIXML", "-I#{include}", "-L#{lib}",
                    "-lecpint", "-o", "example"
```

File: Formula/libecpint.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    cp pkgshare/"hydrogen.bas", testpath
+    cp pkgshare/"iodine.bas", testpath
```
Comment (carlocab) [inline Formula/libecpint.rb:34]:
```suggestion
    cp [pkgshare/"hydrogen.bas", pkgshare/"iodine.bas"], testpath
```

**Comment by junghans** [comment]
@carlocab for some reason there own example core dumped, so going with a different test now.

**Comment by junghans** [comment]
@carlocab this is ready for your review now!

**Comment by junghans** [comment]
Or @chenrui333 that might be one for you, too?

**Review by chenrui333 — APPROVED** [review]
LGTM

**Review by p-linnane — APPROVED** [review]

**Pull request merged**